src/styles/main.scss

```diff
@@ -1,3 +1,203 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
```

Comment:
Not fixed 
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/476a0be1-a2ae-45b5-810f-6dc2b94ef03c)